models/linkedin_ads__account_report.sql

```diff
@@ -0,0 +1,50 @@
+with account as (
+
+    select *
+    from {{ var('account_history') }}
+    where valid_to is null -- get latest 
```

Comment:
Once we update the source package to leverage the `is_most_recent` logic instead, we will want to update this to reflect that change.